runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -1776,6 +1776,70 @@ benchmarks! {
             );
         }
 
+    activate_amm {
```

Comment:
Consider adding a comment about why member channel all max doesn't make op. more expensive

Reply:
curator channel operations seems to be more expensive due to permission checking